utils/wasm-builder/src/wasm_project.rs

```diff
@@ -467,16 +545,16 @@ fn build_project(project: &Path, default_rustflags: &str, cargo_cmd: CargoComman
 		build_cmd.arg("--color=always");
 	}
 
-	if is_release_build() {
-		build_cmd.arg("--release");
-	};
+	let profile = Profile::detect(project);
```

Comment:
Before we always build the wasm in release and only debug if directly requested. Imo we should keep the behavior, because debug is too slow in wasm.

Reply:
Ahh okay I wasn't sure if that was actually intended: The doc of the env variable said that it would use the main build profile but in reality it was using `release` all the time if not overwritten.

I will change it back to default to `release`  for `debug` main builds and change the doc accordingly.

Reply:
Ty :)